test/functional/finalization_withdraw.py

```diff
@@ -59,6 +59,7 @@ def run_test(self):
         finalizer2 = self.nodes[2]
 
         self.setup_stake_coins(*self.nodes)
+        assert_equal(finalizer1.getbalance(), Decimal('10000'))
```

Comment:
I used the hardcoded number to have a better view of how the value changed (increased/decreased).  I agree that using hardcoded numbers in the working code is not a good idea but tests are used to analyze the specification and in this case, I think it's helpful to see concrete numbers instead of computing yourself the outcome.

I'll change to a variable if this is smth that team disagrees.